src/components/TeamSection/Team.tsx

```diff
@@ -3,26 +3,38 @@ import Profile from "./Profile";
 import * as S from "./Team.styles";
 import "./Team.styles.ts";
 import { type TeamMember } from "./TeamInformation";
-import { executiveTeam } from "./TeamInformation";
+import { executiveTeam, projectManagers, alumniTeam } from "./TeamInformation";
```

Comment:
I'd combine these two imports - makes it a bit more compact and manageable as they're both from the same file.